src/modules/pythonmonkey/pythonmonkey.cc

```diff
@@ -623,5 +652,23 @@ PyMODINIT_FUNC PyInit_pythonmonkey(void)
     return NULL;
   }
 
+  // initialize FinalizationRegistry of JSFunctions to Python Functions
```

Comment:
The callback functionRegistryCallback used to decref the pyObject never gets called during the tests

Reply:
Are you sure? I'm not able to replicate this, the function is called many times during the tests for me.

Reply:
you just did a successful demo

Reply:
not sure what I had wrong but it does indeed work! Nice!